### PR TITLE
Expose setId in FeatureCollector [#377]

### PR DESCRIPTION
**Pull request by bdon**
To aid in setting OSM IDs in #377

### CODE REVIEW

File: planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java [hunk 1]
```diff
@@ -240,6 +240,11 @@ public long getSourceId() {
       return sourceId;
     }
 
+    public Feature setSourceId(long sourceId) {
+      this.sourceId = sourceId;
+      return this;
+    }
+
```
Comment (msbarry) [inline planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java:247]: let's change this to `setId`

Comment (bdon) [reply planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java:247]: should we deprecate `getSourceId` and add `getId` ?

Comment (msbarry) [reply planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java:247]: We _could_ but I don't see any references to `getSourceId` in the code, so probably safe to remove.

**Comment by bdon** [comment]
ptal

File: planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java (hunk 1 above)
Comment (bdon) [reply planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureCollector.java]: where you usually interact with `FeatureCollector` you also have access to the `sourceFeature`, right? so might be redundant to have a `getSourceId` that delegate to the `FeatureCollector`'s `source` when you can call `source.id()`

**Pull request merged**